java/arcs/core/data/testdata/BUILD

```diff
@@ -15,6 +15,12 @@ filegroup(
     ),
 )
 
+arcs_kt_gen(
```

Comment:
will it also work for splitting the recipe doing ingestion and a recipe doing consumption, and then importing ingestion manifest from the consumption manifest?

Reply:
I'm working in another branch on this -- wrt recipe2plan. This is more of a fix for arcs_kt_schema in the "master genrule".

Reply:
Ok, thanks!